src/NUnitFramework/framework/Internal/Execution/SimpleWorkItem.cs

```diff
@@ -153,6 +154,10 @@ private TestCommand MakeTestCommand()
                     command = new TimeoutCommand(command, timeout);
 #endif
 
+                // Add wrappers for repeatable tests after timeout so the timeout is reset on each instance
```

Comment:
I apologize for not catching earlier... the term `instance` isn't as clear as saying `on each repeat`.